packages/node_modules/@ciscospark/spark-core/src/interceptors/redirect.js

```diff
@@ -60,6 +62,26 @@ export default class RedirectInterceptor extends Interceptor {
 
       return this.spark.request(options);
     }
+    else if (response.headers
+      && response.body
+      && response.body.errorCode === LOCUS_REDIRECT_ERROR
+      && response.body.location) {
+      options = clone(options);
+      if (response.options && response.options.qs) {
+        const newUrl = response.body.location.split('?');
+        options.uri = newUrl[0]; // params are already present in the qs
```

Comment:
for POST

Reply:
I would actually add these as comments to the file itself.